projects/npm-tools/packages/npm-scripts/__fixtures__/webpack/tweakWebpackConfig/npmscripts.config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const os = require('os');
+
+module.exports = {
+	build: {
+		temp: os.tmpdir(),
```

Comment:
This is needed so that test don't leave files hanging around. I could as well delete them, but still, if the tests failed or where interrupted the source tree would have been polluted, so better move temp things out.

Reply:
One thing to bear in mind about `tmpdir()` is that it might not provide a very good location. For example, on Linux, is is going to return `/tmp` which is just a gigantic bucket shared by all users. On macOS it will at least return something that is user-specific (eg. `'/var/folders/xx/huge-long-string-of-random-digits/x`) and which I think is different on every boot.

If you want each test run to have an independent space where it can write temporary files to without collisions (with other users, or with other runs by the same user), you need to combine `tmpdir()` with `fs.mkdtempSync()` like we do in a number of places in this repo; for [example](https://github.com/liferay/liferay-frontend-projects/blob/8954af4210985cc17eb0da7a2bd30026809585cc/projects/npm-tools/packages/npm-scripts/test/utils/minify.js#L19):

```js
const temp = fs.mkdtempSync(path.join(os.tmpdir(), 'blah-'));
```

Reply:
I'll push a commit for this. In a regular desktop it is very rare that parallel tests ever run, but in CI/QA environments it could happen, you never know...

Reply:
done